server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -131,8 +132,18 @@ public Repo<Manager> call(long tid, Manager manager) throws Exception {
             Text cq;
 
             if (key.getColumnFamily().equals(DataFileColumnFamily.NAME)) {
-              final StoredTabletFile oldTabletFile = StoredTabletFile.of(key.getColumnQualifier());
-              String oldName = oldTabletFile.getFileName();
+              StoredTabletFile exportedRef;
+              var dataFileCQ = key.getColumnQualifier().toString();
+              if (tableInfo.exportedVersion == null || (tableInfo.exportedVersion < VERSION
+                  && StoredTabletFile.fileNeedsConversion(dataFileCQ))) {
```

Comment:
This behavior is very tied to a specific version.  I would make that more explicit as follows with a new constant `VERSION_2`.  If in the future the `VERSION` changes this code will still behave the same w.r.t. the specific version where the change in behavior happened.  It also makes it easier to follow the code over time and know what specific version we care about in this if stmt.

```suggestion
              if (tableInfo.exportedVersion == null || tableInfo.exportedVersion < VERSION_2) {
                  checkState(!StoredTabletFile.fileNeedsConversion(dataFileCQ));
```

Reply:
Added CURR_VERSION and explicit VERSION_2 in 4d136c4c87